src/widget/wmainmenubar.cpp

```diff
@@ -155,9 +155,10 @@ void WMainMenuBar::initialize() {
     connect(this, &WMainMenuBar::internalLibraryScanActive, pLibraryRescan, &QAction::setDisabled);
     pLibraryMenu->addAction(pLibraryRescan);
 
+    //: "Engine DJ" must not be translated
 #ifdef __ENGINEPRIME__
-    QString exportTitle = tr("E&xport Library to Engine Prime");
-    QString exportText = tr("Export the library to the Engine Prime format");
+    QString exportTitle = tr("E&xport Library to Engine DJ");
```

Comment:
Is the `&` expected there?

Reply:
& marks the letter that is used for the mnemonic / keyboard shortcut.
It's been there before, so why remove it?

Though we may add a translator hint for those, too.
Will do that once I clean up this branch in the next days.

Reply:
Na, I think I won't. let's add that when we notice issues with that (and we haven't seen any until now AFAIR)